chartlets.js/packages/lib/src/plugins/mui/Divider.tsx

```diff
@@ -0,0 +1,39 @@
+import type { ElementType } from "react";
+import MuiDivider from "@mui/material/Divider";
+
+import type { ComponentState, ComponentProps } from "@/index";
+import { Children } from "@/index";
+
+interface DividerState extends ComponentState {
+  orientation?: "horizontal" | "vertical";
+  variant?: "fullWidth" | "inset" | "middle";
+  flexItem?: boolean;
+  textAlign?: "left" | "center" | "right";
+  component?: ElementType<Element>;
+}
+
+interface DividerProps extends ComponentProps, DividerState {}
+
+export const Divider = ({
+  id,
+  style,
+  orientation,
+  variant,
+  flexItem,
+  textAlign,
+  children: nodes,
+  onChange,
+}: DividerProps) => {
+  return (
+    <MuiDivider
+      id={id}
+      style={style}
+      orientation={orientation}
+      variant={variant}
+      flexItem={flexItem}
+      textAlign={textAlign}
+    >
+      <Children nodes={nodes} onChange={onChange} />
```

Comment:
Since the nodes are never undefined, because we send an empty array instead, a gap appears in the divider which is not an expected behaviour. The children code can be modified as below
```suggestion
      {nodes && nodes.length && <Children nodes={nodes} onChange={onChange} />}
```